docs/data-product-studio/data-quality/failed-events/monitoring-failed-events/ui/index.md

```diff
@@ -29,12 +41,39 @@ By selecting a particular error you are able to get more detail:
 
 Example failed event detail view.
 
-The detailed view shows the error message as well as other useful meta data (when available) like app_id as an example to help quickly diagnose the source and root cause of the error, as well as a bar chart indicating how the volume of failures varies over time for this particular failed event.
+The detailed view shows the error message as well as other useful metadata (when available) like app_id as an example to help diagnose the source and root cause of the error, as well as a bar chart indicating how the volume of failures varies over time for this particular failed event.
+
+# Data Quality Dashboard
+
+## Architecture
+
+The discussion of architecture is important as it highlights the trade-offs between the two ways of monitoring failed events. As mentioned above, the Data Quality Dashboard is a feature that allows customers to see failed events directly from their warehouse. To achieve that, the customer's browser connects directly to an API running within customer infrastructure, such that no failed event information flows through Snowplow. The aforementioned API is a simple proxy that connects to the customer's warehouse and serves the failed events to the customer's browser. The connection is fully secure, using an encrypted channel (HTTPS) and authenticating/authorizing customers via the same mechanism used by the Console.
```

Comment:
```suggestion
The Data Quality Dashboard allows you to see failed events directly from your warehouse. Your browser connects directly to an API running within your infrastructure, such that no failed event information flows through Snowplow. The discussion of architecture is important as it highlights the trade-offs between the two ways of monitoring failed events. The aforementioned API is a simple proxy that connects to your warehouse and serves the failed events to your browser. The connection is fully secure, using an encrypted channel (HTTPS) and authenticating/authorizing via the same mechanism used by Console.
```

Reply:
I've edited this one but the whole page needs updating: the reader is the customer, we are speaking to them directly